app/src/main/res/values/dimens.xml

```diff
@@ -34,6 +34,18 @@
   <dimen name="font_scale_content_large_text_view_size">24dp</dimen>
   <dimen name="font_scale_content_text_size">20dp</dimen>
 
+  <!-- Resume Lesson Fragment Font Scale -->
+  <dimen name="resume_lesson_fragment_font_scale_content_small_text_size">12.8dp</dimen>
+  <dimen name="resume_lesson_fragment_font_scale_content_medium_text_size">16dp</dimen>
+  <dimen name="resume_lesson_fragment_font_scale_content_large_text_size">19.2dp</dimen>
+  <dimen name="resume_lesson_fragment_font_scale_content_extra_large_text_size">22.4dp</dimen>
+
+  <!-- Revision Card Fragment Fragment Font Scale -->
+  <dimen name="revision_card_fragment_font_scale_content_small_text_size">12.8dp</dimen>
+  <dimen name="revision_card_fragment_font_scale_content_medium_text_size">16dp</dimen>
+  <dimen name="revision_card_fragment_font_scale_content_large_text_size">19.2dp</dimen>
+  <dimen name="revision_card_fragment_font_scale_content_extra_large_text_size">22.4dp</dimen>
```

Comment:
I suspect that the tests will fail if you use a device with a different screen size, therefore point (3) still stands. We don't use decimals in dimensions.

I also think that it is okay to modify the existing font scale to sp, even though it may result in changing some other font scale utility tests. You need to verify that the UI still scales the way it should, but I don't foresee any problems with that.

Reply:
I don't think the test will fail even with a different screen size, because I have tried running the test on multiple devices.